### PR TITLE
add aws help scripts

### PR DESCRIPTION
**Pull request by zzhuang6**

### CODE REVIEW

File: cloud/aws/README [hunk 1]
```diff
@@ -0,0 +1,22 @@
+The script in this directory are to help user to get information of ec2 and run commands in all ec2 cluster.
```
Comment (gaolk) [inline cloud/aws/README]: this scripts
users
in a cluster of ec2s

File: cloud/aws/README [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+=============================================================================================================================
+
+aws_cli.sh and aws_config
+--------------------------
+These two files are used to get public ip/dns and private ip of ec2 based on given filters.
+To get the info:
+1. put your aws key and secret to aws_config file
+2. source aws_cli.sh
+3. run get_all_public_ip or other command
```
Comment (gaolk) [inline cloud/aws/README]: commands

File: cloud/aws/README [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+run_all_ec2.sh
+----------------
+This file is used to run command/script on all ec2 instance.
```
Comment (gaolk) [inline cloud/aws/README]: This scripts  .... instances

File: cloud/aws/aws_cli.sh [hunk 4]
```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+######################################
+## Helps to get ec2 info using awscli
+##
+## Requirement: aws command installed
+##     ubtunu: sudo apt-get install awscli
```
Comment (gaolk) [inline cloud/aws/aws_cli.sh]: ubuntu

File: cloud/aws/README [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+To run the command:
+1. put all public ip/dns to pip.txt file (use aws_cli.sh to get the ips)
+2. run bash run_all_ec2.sh "your_command"
```
Comment (gaolk) [inline cloud/aws/README]: 0. need to change the pem and username in run_all_ec2.sh

**Comment by zzhuang6** [comment]
@gaolk fixed

**Pull request merged**